jetcd-core/src/main/java/io/etcd/jetcd/resolver/IPNameResolver.java

```diff
@@ -61,21 +66,37 @@ public IPNameResolver(URI targetUri) {
         this.lock = new Object();
         this.targetUri = targetUri;
         this.authority = targetUri.getAuthority() != null ? targetUri.getAuthority() : SCHEME;
-        this.addresses = Stream.of(targetUri.getPath().split(","))
-            .map(address -> {
-                return address.startsWith("/")
-                    ? address.substring(1)
-                    : address;
-            })
-            .map(address -> {
+
+        List<String> defaultAuthorities = Arrays.stream(this.authority.split(",")).collect(Collectors.toList());
```

Comment:
don't think is is needed to have a list here , using a String[] should be enough